frontend/src/util/helper.js

```diff
@@ -131,3 +131,46 @@ export const sortWorkspaceList = (a, b) => {
   if (b.label > a.label) return -1
   return 0
 }
+
+export const toggleFavicon = (hasNewNotification) => {
+  const originalHrefAttribute = 'originalHrefAttribute'
+
+  document.getElementsByClassName('tracim__favicon').forEach(favicon => {
+    if (!hasNewNotification) {
+      favicon.href = favicon.getAttribute(originalHrefAttribute)
+      favicon.removeAttribute(originalHrefAttribute)
+      return
+    }
+    const faviconSize = favicon.sizes[0].split('x')[0]
+
+    const canvas = document.createElement('canvas')
+    canvas.width = faviconSize
+    canvas.height = faviconSize
+
+    const context = canvas.getContext('2d')
+    const img = document.createElement('img')
+    img.src = favicon.href
+
+    img.onload = () => {
+      // INFO - GM - 2020/08/18 - Draw Original Favicon as Background
+      context.drawImage(img, 0, 0, faviconSize, faviconSize)
+
+      // INFO - GM - 2020/08/18 - Draw Notification Circle
+      context.beginPath()
+      context.arc(
+        canvas.width - faviconSize / 6,
+        canvas.height - faviconSize / 6,
+        faviconSize / 6, 0,
```

Comment:
Why 6? can you put it in a constant?

Reply:
I choose 6 arbitrarily to get the rendering I wanted, Yes you are right, it is the circle size